insights/parsers/tests/test_numeric_user_group_name.py

```diff
@@ -0,0 +1,64 @@
+from insights.parsers.numeric_user_group_name import NumericUserGroupName
+from insights.tests import context_wrap
+
+ZERO = "0"
+NONZERO = "3"
+UNSTRIPPED_ZERO = """
+
+0
+"""
+UNSTRIPPED_NONZERO = """
+
+3
+
+"""
+
+
+# Deliberately testing various ways of comparing the value, some of which are not liked by flake8.
```

Comment:
Actually, I think we do not need any of these equality / inequality tests. I think a one method which returns number of users / groups starting with number is the only thing we need there.